### PR TITLE
WRP-3069: Update package dependencies to fix security vulnerabilities

### PR DESCRIPTION
**Pull request by andreitirlalge**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [ ] I have run automated CLI testing and it is passed
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Updated dependencies to the latest minor version.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
All dependencies are now updated to latest version.

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRP-3069

### Comments
Enact-DCO-1.0-Signed-off-by: Andrei Tirla andrei.tirla@lgepartner.com

### CODE REVIEW

**Review by daniel-stoian-lgp — APPROVED** [review]
LGTM

File: package.json [hunk 1]
```diff
@@ -20,17 +20,17 @@
...
-    "@typescript-eslint/parser": "^5.44.0",
+    "@typescript-eslint/eslint-plugin": "^5.48.1",
+    "@typescript-eslint/parser": "^5.48.1",
     "babel-preset-react-app": "^10.0.1",
-    "eslint": "^8.28.0",
+    "eslint": "^8.31.0",
     "eslint-plugin-enact": "^1.0.1",
-    "eslint-plugin-jest": "^27.1.6",
-    "eslint-plugin-jsx-a11y": "^6.6.1",
+    "eslint-plugin-jest": "^27.2.1",
```
Comment (daniel-stoian-lgp) [inline package.json:32]: this dependency should be left untouched in this PR since we agreed to downgrade it in another user story

**Review by daniel-stoian-lgp — COMMENTED** [review]
please check my comment

**Review by daniel-stoian-lgp — APPROVED** [review]

**Comment by adrian-cocoara-lgp** [comment]
Please update these dependencies:
eslint                   ^8.31.0  →  ^8.32.0
eslint-plugin-jsx-a11y    ^6.7.0  →   ^6.7.1
eslint-plugin-react     ^7.31.11  →  ^7.32.0

**Review by adrian-cocoara-lgp — APPROVED** [review]

**Comment by hong6316** [comment]
to @andreitirlalge 
Please pinning eslint-plugin-react to ^7.32.0 -> 7.31.11
Reason: breaking changes occurred(related WRP-6801)
And make a new ticket about updating `eslint-plugin-react` to v7.32.x

**Review by hong6316 — APPROVED** [review]
👍

**Pull request merged**